From 9d3629f7a6edce20990ca75d879097abe62bc8a7 Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Sun, 16 Feb 2020 02:51:52 +0000
Subject: [PATCH] drop unsupported keycodes

Jun 14 00:55:54 NUC xorg-launch[722]: The XKEYBOARD keymap compiler (xkbcomp) reports:
Jun 14 00:55:54 NUC xorg-launch[722]: > Warning:          Unsupported high keycode 569 for name <I569> ignored
Jun 14 00:55:54 NUC xorg-launch[722]: >                   X11 cannot support keycodes above 255.
Jun 14 00:55:54 NUC xorg-launch[722]: >                   This warning only shows for the first high keycode.
Jun 14 00:55:54 NUC xorg-launch[722]: > Internal error:   Could not resolve keysym XF86RotationLockToggle
Jun 14 00:55:54 NUC xorg-launch[722]: Errors from xkbcomp are not fatal to the X server
---
 keycodes/evdev | 3 ---
 symbols/inet   | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/keycodes/evdev b/keycodes/evdev
index 8ed0fa9..f7d5c7b 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -284,11 +284,8 @@ default xkb_keycodes "evdev" {
 	<I254> = 254;	// #define KEY_WWAN                246
 	<I255> = 255;	// #define KEY_RFKILL              247
 
-	<I380> = 380;   // #define KEY_FULL_SCREEN         372
	<I442> = 442;   // #define KEY_DOLLAR              434
	<I443> = 443;   // #define KEY_EURO                435
-	<I569> = 569;   // #define KEY_ROTATE_LOCK_TOGGLE  561
-
 	// Fake keycodes for virtual keys
 	<LVL3> =   92;
 	<MDSW> =   203;
diff --git a/symbols/inet b/symbols/inet
index 2a5ab74..0fb78cd 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -218,11 +218,8 @@ xkb_symbols "evdev" {
     key <I254>   {      [ XF86WWAN              ]       }; // KEY_WWAN
     key <I255>   {      [ XF86RFKill            ]       }; // KEY_RFKILL
 
-    key <I380>  {       [ XF86FullScreen         ]       };
     key <I442>  {       [ dollar                 ]       };
     key <I443>  {       [ EuroSign               ]       };
-    key <I569>  {       [ XF86RotationLockToggle ]       };
-
     key <FK13>   {      [ XF86Tools         ]       };
     key <FK14>   {      [ XF86Launch5       ]       };
     key <FK15>   {      [ XF86Launch6       ]       };
-- 
2.7.4

